sway-core/src/semantic_analysis/namespace.rs

```diff
@@ -164,7 +164,7 @@ impl Namespace {
 #[derive(Clone, Debug, Default)]
 pub(crate) struct TraitMap {
     // This cannot be a HashMap because of how TypeInfo's are handled.
-    //
+    // 
```

Comment:
Hmm the extra space doesn't seem to match how other comments look
```suggestion
    //
```